src/Webapi/Webapi__Url.re

```diff
@@ -21,7 +21,7 @@ module URLSearchParams = {
 type t;
 
 [@bs.new] external make: string => t = "URL";
-[@bs.new] external makeWithBase: (string, string) => t = "URL";
+[@bs.new] external makeWithBase: (string, ~base: string) => t = "URL";
```

Comment:
Since this is a breaking change, I think it's better to make a new function and deprecate this one. The new one can be `makeWith(string, ~base)`.

Reply:
Done.